src/app/app.component.html

```diff
@@ -1,2 +1,2 @@
 <h1>NgWig Demo</h1>
-<ng2wig></ng2wig>
+<ng2wig plugins="demo"></ng2wig>
```

Comment:
So, how the user will be able to select which plugins to load? Plugins attribute is used here to load the component in question dynamically. Alternatively, he must add the HTML selector of the component manually. No?